pkg/api/v1/types.go

```diff
@@ -1312,6 +1312,15 @@ type NodeSpec struct {
 	Unschedulable bool `json:"unschedulable,omitempty" description:"disable pod scheduling on the node; see http://releases.k8s.io/HEAD/docs/admin/node.md#manual-node-administration"`
 }
 
+type KubeletPorts struct {
+	// Port on which Kubelet is listening
+	KubeletPort uint `json:"kubeletPort" description:"port on which Kubelet is listening"`
```

Comment:
While uint might technically be correct, we don't use uint in the API anywhere. We use int for ports.